week02/largestNumber.cpp

```diff
@@ -0,0 +1,23 @@
+//TC: O(NlogN)
+//MC: O(N)
+class Solution {
+public:
+    string largestNumber(vector<int>& nums) {
+        vector<string>numString;
+        
+        for(int i: nums){
+            numString.emplace_back(to_string(i));
+         }
+        
+        sort(numString.begin(), numString.end(), [](string a, string b){
+                return (a+b)>(b+a);
+        });
```

Comment:
In the lambda expression, string a and string b are being passed as values. That means, while comparing, each time there will be an extra copy created just for comparison. Very time-costly operation. You could have passed the reference only.